src/js/queries/allServicePagesQuery.js

```diff
@@ -10,12 +10,16 @@ const allServicePagesQuery = `
             id
             slug
             text
+            theme {
```

Comment:
same question as above, I forget the topic to theme relationship. 1 to 1? If topics can have multiple themes, this may get goofy. Same if services can be under multiple topics, breadcrumbs can get wild.